clone/cmd/sumdbclone/docker-compose.yaml

```diff
@@ -0,0 +1,32 @@
+version: "3.1"
+
+services:
+  db:
+    image: mariadb
+    restart: always
+    environment:
+      MYSQL_ROOT_PASSWORD: example
+      MYSQL_DATABASE: sumdb
+      MYSQL_USER: sumdb
+      MYSQL_PASSWORD: letmein
+    healthcheck:
+      test: mysql --user=$$MYSQL_USER --password=$$MYSQL_PASSWORD --silent --execute "SHOW DATABASES;"
+      interval: 3s
+      timeout: 2s
+      retries: 5
+
+  clone:
+    build: 
+      context: ../../..
+      dockerfile: ./clone/cmd/sumdbclone/Dockerfile
+    command: [
+      "--alsologtostderr",
+      "--v=2",
+      "--mysql_uri=sumdb:letmein@tcp(db:3306)/sumdb",
```

Comment:
Is it possible to have common vars (or something) shared across both services so the DB username/password only needs to be set once?

Reply:
Pulled these things out to a .env file which does allow for variable expansion in other services.